bundles/org.openhab.binding.ring/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file can be removed

Reply:
This is only in to have backwards compatibility with 3.4 for the marketplace.  Same as a few of the lines in pom.xml.  I will pull these out before merge.  This is mostly so I don't have to maintain multiple branches while testing as the bulk of the users are still 3.4.